cli/mne_bids_cp.py

```diff
@@ -0,0 +1,61 @@
+"""Rename files (making a copy) and update their internal pointers.
+
+example usage: $ mne_bids cp --input myfile.vhdr --output sub-01_task-test.vhdr
+"""
+# Authors: Stefan Appelhoff <stefan.appelhoff@mailbox.org>
+#
+# License: BSD (3-clause)
+import mne_bids
+from mne_bids.copyfiles import (copyfile_brainvision, copyfile_eeglab,
+                                copyfile_ctf)
+
+
+def run():
+    """Run the cp command."""
+    from mne.commands.utils import get_optparser
+
+    accepted_formats_msg = ('Accepted formats are BrainVision (.vhdr), '
+                            'EEGLAB (.set), CTF (.ds).')
+
+    parser = get_optparser(__file__, usage="usage: %prog -i INPUT -o OUTPUT",
+                           prog_prefix='mne_bids',
+                           version=mne_bids.__version__)
+
+    parser.add_option('-i', '--input', dest='input',
+                      help=('Path to the input file. {}'
+                            .format(accepted_formats_msg)), metavar='INPUT')
+
+    parser.add_option('-o', '--output', dest='output',
+                      help=('Path to the output file. MUST be same format '
+                            'as input file.'), metavar='OUTPUT')
+
+    parser.add_option('-v', '--verbose', dest="verbose",
+                      help='Set logging level to verbose', action="store_true")
+
+    opt, args = parser.parse_args()
+    opt_dict = vars(opt)
+
+    # Check the usage and raise error if invalid
+    if len(args) > 0:
+        parser.error('Do not specify arguments without flags. Found: "{}".\n'
+                     'Did you forget to provide -i and -o?'
+                     .format(args))
```

Comment:
You should not throw an error. Rather it should display the usage and help.

Reply:
Why should it not throw an error?

This is the current output:

```Shell
(mne) stefanappelhoff@arc-lin-004309:~/Desktop/bids/mne-bids$ mne_bids cp a b
Usage: mne_bids cp -i INPUT -o OUTPUT

mne_bids cp: error: Do not specify arguments without flags. Found: "['a', 'b']".
Did you forget to provide -i and -o?

```

Reply:
Because that's how most command line utils work. Try doing:

```sh
$ git
$ git mv
```

Reply:
```
(base) stefanappelhoff@arc-lin-004309:~$ git mv
fatal: not a git repository (or any of the parent directories): .git

```

Reply:
Do it in a git folder :)

Reply:
But okay I can live with this. Resolving